src/main/java/it/gov/pagopa/payhub/auth/service/exchange/AccessTokenBuilderService.java

```diff
@@ -43,8 +46,11 @@ public AccessTokenBuilderService(
 
     public AccessToken build(){
         Algorithm algorithm = Algorithm.RSA512(rsaPublicKey, rsaPrivateKey);
+        Map<String, Object> headerClaims = new HashMap<>();
+        headerClaims.put("typ", ACCESS_TOKEN_TYPE);
```

Comment:
is this in conflicth with row 54?

Reply:
there are 2 typ, first one into header and the second one into payload.